tests/data/datasets/dataset1/sequenceAnnotations/sacCerTest.gff3

```diff
@@ -4,8 +4,8 @@
 # Saccharomyces cerevisiae S288C genome
 #
 # has empty Alias with multiple values:
-chrI	SGD	chromosome	1	230208	.	.	.	ID=chrI;dbxref=NCBI:NC_001133
 chrI	SGD	repeat_region	1	62	.	-	.	ID=TEL01L-TR;Name=TEL01L-TR;Note=Terminal%20stretch%20of%20telomeric%20repeats%20on%20the%20left%20arm%20of%20Chromosome%20I;dbxref=SGD:S000028864
+chrI	SGD	chromosome	1	230208	.	.	.	ID=chrI;dbxref=NCBI:NC_001133
```

Comment:
I am not sure why did this happen... it is all good just weird

Reply:
I changed this

Reply:
Update: I believe it is related to the order of the table. We want to make sure that the first row of data has "+" or "-" defined, which is why it was moved.